src/components/Bullet.tsx

```diff
@@ -258,13 +265,13 @@ const Bullet = ({
         // calculate position of thought for different font sizes
         // Table column 1 needs more space between the bullet and thought for some reason
         marginLeft: (fontSize - 9) * 0.5 - 11 - extendClickWidth - (isTableCol1 ? fontSize / 4 : 0),
-        marginBottom: -extendClickHeight - 2,
+        marginBottom: extendClickHeight - 2,
```

Comment:
Since the Bullet is now absolutely positioned, do we need `marginBottom` at all?

Regardless, it does not make sense to reverse the sign here: it is meant to exactly compensate for the `paddingBottom`, i.e. increase the clickable area without increasing the height. Perhaps something else necessitates a change to the margin, but `extendClickHeight` should only be used if it's compensating for paddingBottom.

Reply:
I think this is unintentional, sorry probably messed with this while undoing the color blocks for bullet